assets/components/contributionPaymentCtas/contributionPaymentCtas.jsx

```diff
@@ -70,19 +69,15 @@ function onCtaClick(isDisabled: boolean, resetError: void => void): Function {
 export default function ContributionPaymentCtas(props: PropTypes) {
 
   const baseClassName = 'component-contribution-payment-ctas';
-  const paypal = (
-    <props.PayPalButton
-      buttonText={`Contribute ${props.currency.glyph}${props.amount} with PayPal`}
-      onClick={props.resetError}
-    />
-  );
-  const stripe = <OneOffCta {...props} />;
 
   if (props.contributionType === 'ONE_OFF') {
     return (
       <div className={classNameWithModifiers(baseClassName, props.isDisabled ? ['disabled'] : [])}>
-        {props.isInPaymentLogosVariant ? paypal : stripe}
-        {props.isInPaymentLogosVariant ? stripe : paypal}
+        <OneOffCta {...props} />
```

Comment:
You might as well swap these buttons around now as that's the layout we want ultimately

Reply:
Applied in 55f2d1a 

It looks like this now:

### support-landing
![picture 764](https://user-images.githubusercontent.com/825398/40356296-87e83652-5db0-11e8-96d6-314aeb6b4b79.png)

### contribution-landing
![picture 765](https://user-images.githubusercontent.com/825398/40356297-88016302-5db0-11e8-814c-9ecd731614bc.png)

Reply:
👍 Perfect!